.travis.yml

```diff
@@ -20,7 +20,7 @@ jdk:
   - oraclejdk12
   - oraclejdk13
 
-script: mvn clean test
+script: mvn -X clean test
```

Comment:
I think this -X flag has been removed in past config files before

Reply:
Updated